roles/install_yamls/tasks/zuul_set_operators_repo.yml

```diff
@@ -15,23 +15,43 @@
 # under the License.
 
 # When using CI (Zuul) to deploy operators and its dependencies with install_yamls,
-#  it may be needed to set operator's repo variable to properly clone PR's
-#  code, instead of getting latest promoted content. This task search for all
-#  modified operators in zuul.items[] and set install_yaml variables.
+# it may be needed to set operator's repo variable to properly clone PR's
+# code, instead of getting latest promoted content. This task search for all
+# modified operators in zuul.items[] and set install_yaml variables.
+
 - name: Create variables with local repos based on Zuul items
   when:
     - zuul is defined
     - "'operator' in zuul_item.project.short_name"
     - "'openstack-k8s-operators' in zuul_item.project.name"
-  vars:
-    _repo_operator_name: "{{ zuul_item.project.short_name | regex_search('(?:openstack-)?(.*)-operator', '\\1') | first }}"
-    _repo_operator_info:
-      - key: "{{ _repo_operator_name | upper }}_REPO"
-        value: "{{ ansible_user_dir }}/{{ zuul_item.project.src_dir }}"
-      - key: "{{ _repo_operator_name | upper }}_BRANCH"
-        value: ""
-  ansible.builtin.set_fact:
-    cifmw_install_yamls_operators_repo: "{{ cifmw_install_yamls_operators_repo | default({}) | combine(_repo_operator_info | items2dict) }}"
-  loop: "{{ zuul['items'] }}"
-  loop_control:
-    loop_var: zuul_item
+  block:
```

Comment:
That is optional. I was hoping to add such thing for easier debugging. Can be removed if necessary